docs/general-faq.md

```diff
@@ -0,0 +1,5 @@
+# General FAQ
+
+<details> <summary> 1. Why is my TechPass account disabled? How do I re-enable it?</summary>
+
+Your TechPass account will be disabled if you have not used your TechPass account for 90 consecutive days. To re-enable it, contact your
```

Comment:
contact your? what follows it?

Reply:
This page is redundant as we already have https://docs.developer.tech.gov.sg/docs/techpass-user-guide/#/support/signinissues. I have commented the text in this page.

Reply:
this looks like an addition of code, rather than being commented out.. if the entire page is redundant, do delete the file instead.

Reply:
done